code/modules/projectiles/firemodes/firemode_remote.dm

```diff
@@ -67,8 +67,7 @@
 
 		update_tethers()
 
-	if (!is_processing)
-		START_PROCESSING(SSfastprocess, src)
```

Comment:
what happens if you try to start processing multiple times?

Reply:
it'll result in a runtime.